source_code/aux_mcu/src/main.c

```diff
@@ -116,10 +118,10 @@ int main(void) {
 
     // Power Manager init
     power_manager_init();
-    
+
     // Clock Manager init
     clock_manager_init();
-
+#if 1
```

Comment:
why?

Reply:
#if 1 --> USB working /* Change it to 0 to have BLE working */

I can update the code to be more read friendly.